data-processing-lib/python/src/data_processing/data_access/data_access.py

```diff
@@ -185,17 +188,19 @@ def sample_input_data(self, n_samples: int = 10) -> dict[str, Any]:
             average table size MB,
             average doc size KB,
             estimated number of docs
+        and number of retries
         """
         # get files to process
-        path_list, path_profile = self.get_files_to_process_internal()
+        path_list, path_profile, retries = self.get_files_to_process_internal()
         # Pick files to sample
         files = self.get_random_file_set(n_samples=n_samples, files=path_list)
         # Read table and compute number of docs and sizes
         number_of_docs = []
         table_sizes = []
         n_tables = 0
         for f in files:
-            table = self.get_table(path=f)
+            table, r = self.get_table(path=f)
+            retries += r
```

Comment:
Ok, so this appears to be the number of retries across ALL files so we lose the details about which file/s is/are a problem.  Do we need that?  Maybe not yet, but at least the doc needs to be clear about what the number is.

Reply:
As mentioned above this is a way to detect extra pressure on S3. We want to see if the pressure is up by analyzing the amount of retries